blog/2015/11/r-code-of-conduct.html

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+root: ../../..
+author: Kara Woo
+title: "R Foundation Announces Code of Conduct Policy"
+date: 2015-11-08
+time: "12:00:00"
+category: []
+---
+<!-- start excerpt -->
+<p>
+  The R Foundation
+  recently <a href="https://stat.ethz.ch/pipermail/r-announce/2015/000591.html">announced</a>
+  that all conferences it supports must have a code of conduct. They encourage
+  other R meetings to adopt codes of conduct as well, stating that:
+</p>
+
+<blockquote>
+  <p>
+    A code of conduct serves two important purposes. Firstly, it sends a clear
+    message to those outside the community that an R conference is a
+    professional and comfortable working environment for all participants.
+    Secondly, it provides a mechanism for reporting and monitoring any incidents
+    of harassment that may occur.
+  </p>
+</blockquote>
+<!-- end excerpt -->
+<p>
+  Acknowledging that a written code of conduct is insufficient without
+  enforcement, the R Foundation has also stipulated that the codes of conduct
+  should be legally and practically enforceable.
+</p>
+
+<p>
+  This announcement came as welcome news to members of the R community who had
+  requested such a policy. Last June many R users formalized this request in
+  an <a href="https://docs.google.com/document/d/1C1oPhup72lPHJXbpyJZNIo1BdCzfxZ_VoiWJWzmufjU/edit?usp=sharing">open
+  letter to the R Foundation</a>.
+</p>
+
+<p>
+  With its new policy, the R Foundation joins a growing number of academic and
+  professional organizations that have instituted codes of conduct for all of
```

Comment:
professional organizations (including Software Carpentry)

and then make a hyperlink around 'Software Carpentry' to "{{page.root}}/conduct.html".